packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -983,6 +992,34 @@ export default class Meeting extends StatelessWebexPlugin {
     return this.members;
   }
 
+  /**
+   * Reference to the mediaMetrics object
+   * @returns {MediaMetrics}
+   * @public
+   * @memberof Meeting
+   */
+  getMediaMetrics() {
+    return this.mediaMetrics;
+  }
+
+  /**
+   * Automatically publishes media metrics data flattened and on a config interval
+   * uses the existing getStats data filter, so no new streams are created
+   * but it is separate than history, so history will not be available
+   * @returns {MediaMetrics}
+   * @public
+   * @memberof Meeting
+   */
+  autoStats() {
+    const automatic = new StatsMetrics(this.attrs, this.options);
```

Comment:
Renaming the var might make this a bit more readable:

```suggestion
    const automaticMetrics = new StatsMetrics(this.attrs, this.options);
```